.bazelrc

```diff
@@ -35,6 +35,10 @@ build --nolegacy_external_runfiles
 # Don't bother building targets which aren't dependencies of the tests.
 test --build_tests_only
 
+# Default changed in bazel 8
```

Comment:
Please rephrase that this is mandatory at the moment.

Reply:
done